tfhe/src/core_crypto/fft_impl/f128_u128/mod.rs

```diff
@@ -0,0 +1,4 @@
+#![allow(clippy::too_many_arguments)]
```

Comment:
i can add it on individual functions, but there's quite a few of them, since the fft functions now take 4 slices instead of 1, and the split fft module additionally splits the inputs and outputs into 2 slices.

what do you think?

Reply:
ok fine, plus it is limited to f128_u128 so ok for me in the end !